flair/datasets/sequence_labeling.py

```diff
@@ -564,6 +564,61 @@ def __init__(
         )
 
 
+class BUSINESS_HUN(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            tag_to_bioes: str = "ner",
+            in_memory: bool = True,
+            document_as_sequence: bool = False,
+            **corpusargs,
+    ):
+        """
+        Initialize the NER Business corpus for Hungarian. The first time you call this constructor it will automatically
+        download the dataset.
+        :param base_path: Default is None, meaning that corpus gets auto-downloaded and loaded. You can override this
+        to point to a different folder but typically this should not be necessary.
+        :param tag_to_bioes: NER by default, need not be changed, but you could also select 'pos' to predict
+        POS tags instead
+        :param in_memory: If True, keeps dataset in memory giving speedups in training.
+        :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+        """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+        
+        # column format
+        columns = {0: "text", 1: "ner"}
+
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = Path(flair.cache_root) / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        hun_ner_path = "https://rgai.sed.hu/sites/rgai.sed.hu/files/business_NER.zip"
+        path_to_zipped_corpus = cached_path(hun_ner_path, Path("datasets") / dataset_name)
+        unpack_file(
+            path_to_zipped_corpus , 
+            data_folder,
+            mode = "zip",
+            keep= True 
+            )
+
+        super(BUSINESS_HUN, self).__init__(
+            data_folder,
+            columns,
+            train_file='hun_ner_corpus.txt',
```

Comment:
Can you also add the line 
`label_name_map={'0': 'O'}, `
here? Most datasets use O ("out") instead of 0 to denote that something is not an entity. This line will map all tags from 0 to O.